packages/contract-addresses/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as _ from 'lodash';
+
+export interface ContractAddresses {
```

Comment:
Hmm.. I'm not sure I agree with renaming it. In this case, I think the kind of information we are passing around (the addresses of deployed contracts) is more important to call out than the underlying data structure (a mapping of contract name to contract address). For similar reasons, it seems awkward to rename `getContractAddressesForNetwork` to `getContractToAddressForNetwork` or rename the `contractAddresses` parameter in the constructor for `ContractWrappers` to `contractToAddress`.

To look at it from another perspective, here are a few examples of how we use the `XToY` naming convention in `@0xproject/types`: 

```
export interface ExportNameToTypedocNames {
    [exportName: string]: string[];
}

export interface ExternalTypeToLink {
    [externalTypeName: string]: string;
}

export interface ExternalExportToLink {
    [externalExport: string]: string;
}
```

All of these cases represent a mapping of *arbitrary*  X to Y, and that's where the naming convention makes the most sense to me. Since we don't know ahead of time exactly what keys the type will hold (and whether other uses of the type will have the same keys), the underlying data structure is more important to call out. In other words, the data structure itself is what defines the type, not the data it holds.

All that said, if you feel strongly about it, I can be convinced.

Reply:
Yeah, I see your point in this case. Thanks for the thoughtful explanation, let's leave it the way it is.